server/src/dtos/asset.dto.ts

```diff
@@ -74,6 +75,10 @@ export class UpdateAssetDto extends UpdateAssetBase {
   @Optional()
   @IsString()
   description?: string;
+
+  @Optional()
+  @IsIn(['1', '2', '3', '4', '5', '6', '7', '8'])
```

Comment:
Might be worth using the enum here

Reply:
done

Reply:
You should use `@IsEnum` and @ApiProperty` for the enum actually. You can look at other usages in the codebase.

Reply:
done